### PR TITLE
feat(filters): convert jQuery to native elements on multiple filters

### PR DESCRIPTION
**Pull request by ghiscoding**

### CODE REVIEW

File: packages/common/src/filters/compoundSliderFilter.ts [hunk 1]
```diff
@@ -250,66 +225,102 @@ export class CompoundSliderFilter implements Filter {
...
+    this.filterInputElm.value = searchTermInput;
+    this.filterInputElm.min = `${minValue}`;
+    this.filterInputElm.max = `${maxValue}`;
+    this.filterInputElm.step = `${step}`;
+    this.filterInputElm.name = this._elementRangeInputId;
+
+    const divContainerFilterElm = document.createElement('div');
+    divContainerFilterElm.className = `form-group search-filter slider-container filter-${columnId}`;
+
+    // this.containerInputGroupElm = $(`<div class="input-group search-filter filter-${columnId}"></div>`);
```
Comment (AnnetteZhang) [inline packages/common/src/filters/compoundSliderFilter.ts:264]: Can this commmented code be removed?

File: packages/common/src/filters/compoundSliderFilter.ts [hunk 2]
```diff
@@ -250,66 +225,102 @@ export class CompoundSliderFilter implements Filter {
...
+      this.filterNumberElm = document.createElement('span');
+      this.filterNumberElm.className = `input-group-text ${this._elementRangeOutputId}`;
+      this.filterNumberElm.textContent = searchTermInput;
+      divGroupAppendElm.appendChild(this.filterNumberElm);
+      this.containerInputGroupElm.appendChild(divGroupAppendElm);
+    }
 
-    this.$filterInputElm.val(searchTermInput);
-    this.$filterInputElm.data('columnId', columnId);
+    // this.filterNumberElm.html(searchTermInput);
```
Comment (AnnetteZhang) [inline packages/common/src/filters/compoundSliderFilter.ts:285]: Can this commmented code be removed?

**Review by AnnetteZhang — COMMENTED** [review]
A couple of minor changes

File: packages/common/src/services/filter.service.ts [hunk 3]
```diff
@@ -117,9 +117,8 @@ export class FilterService {
 
   dispose() {
     // unsubscribe all SlickGrid events
-    if (this._eventHandler && this._eventHandler.unsubscribeAll) {
-      this._eventHandler.unsubscribeAll();
-    }
+    this._eventHandler.unsubscribeAll();
```
[the remark this answers is not in the document]
Comment (ghiscoding) [reply packages/common/src/services/filter.service.ts:120]: don't need because the EventHandler is initiated in the constructor and is always filled

**Review by AnnetteZhang — APPROVED** [review]

**Pull request merged**